lib/active_record_doctor/detectors/missing_unique_indexes.rb

```diff
@@ -100,6 +106,20 @@ def conditional_validator?(validator)
         (validator.options.keys & [:if, :unless, :conditions]).present?
       end
 
+      def has_and_belongs_to_many_without_indexes # rubocop:disable Naming/PredicateName
+        each_model do |model|
+          each_association(model, type: :has_and_belongs_to_many, has_scope: false) do |association|
+            join_table = association.join_table
+            next if config(:ignore_join_tables).include?(join_table)
+
+            columns = [association.foreign_key, association.association_foreign_key]
```

Comment:
I think we should also check for a unique index on the two columns in reverse order as both `(a, b)` and `(b, a)` could do the job.

Reply:
This is already handled by `unique_index?` method.

Reply:
You're right 👍🏻